go/vt/vtctl/reparentutil/util_test.go

```diff
@@ -1218,3 +1218,105 @@ func Test_getTabletsWithPromotionRules(t *testing.T) {
 		})
 	}
 }
+
+func Test_GetBackupCandidates(t *testing.T) {
```

Comment:
naming nit: (the other ones have the underscore because otherwise they would read as `TestgetTabletsWithPromotionRules`, for example)
```suggestion
func TestGetBackupCandidates(t *testing.T) {
```